src/pages/our_ambassadors.astro

```diff
@@ -314,6 +314,19 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
         Moritz E. Beber...
       </AmbassadorCard>
 
+      <AmbassadorCard
+        name="Muntadher Jihad"
+        img="munji.png"
+        country="iq"
```

Comment:
Hi Muntadher. You live in the UK, right? The country flag is supposed to highlight your location, not your origin, so that people who want to reach out to you know what country you're located.

Reply:
yes, I am in UK. thank you for the correction